### PR TITLE
chore(internals)!: remove legacy photonResolver and "provider=photonjs" handling

### LINKED ISSUE #19319: Prisma Client: remove legacy `photonResolver` and `provider=photonjs` handling
Prisma Client used to be called "Photon" and we want to remove the error associated with that rename since it has been a few years now.

PR: https://github.com/prisma/prisma/pull/19193

### PR DESCRIPTION
**Pull request by aqrln**
Closes: https://github.com/prisma/prisma/issues/19319

### CODE REVIEW

File: packages/internals/src/get-generators/generatorResolvers/generatorResolvers.ts [hunk 1]
```diff
@@ -19,6 +18,5 @@ export type GeneratorResolvers = {
  * as written in the schema, as well as finding their output paths.
  */
 export const generatorResolvers: GeneratorResolvers = {
```
Comment (SevInf) [inline packages/internals/src/get-generators/generatorResolvers/generatorResolvers.ts:20]: I guess we could also remove this map completely now and just use `prismaClientResolver` directly?

Comment (aqrln) [reply packages/internals/src/get-generators/generatorResolvers/generatorResolvers.ts:20]: Yes, good point.

**Comment by janpio** [comment]
ping @Jolg42 about the expanded scope of also removing the `photonjs` handling

**Comment by Jolg42** [comment]
Thanks @janpio, no issue with that for me, only that we need to document it.

**Review by Jolg42 — APPROVED** [review]

**Comment by aqrln** [comment]
There was an unresolved review comment about removing the map and using `prismaClientResolver` directly which I completely forgot about. Should I open a follow-up PR?

**Comment by Jolg42** [comment]
Sorry about that, I don't think it's super important / has much value but of course can be done anytime now if you feel like it.

**Comment by aqrln** [comment]
No worries at all, just asking to know if this is something we still want to do.

**Pull request merged**